apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -2551,6 +2552,67 @@ suite('conversation.js >', function() {
             });
           }).then(done, done);
         });
+
+        test('second chunk ready, then scroll up', function(done) {
+          var nbMessages = firstChunkSize + 2 * chunkSize;
```

Comment:
It's not a big deal since it's clear what this variable is about if one reads few lines below :) I thought about ```numberOfMessages``` or ```messageCount```